testing_exercise/testing.js

```diff
@@ -0,0 +1,62 @@
+function replaceWith(string, ltrOld, ltrNew) {
+	var newString = "";
+	for(var i=0; i<string.length; i++) {
```

Comment:
space after the for, space between = sign, space between the < sign:

for (var i = 0; i < string.length; i++) {